### PR TITLE
Add Logintest

### PR DESCRIPTION
**Pull request by AngieCristina**
**Implementation Notes**
- Added Login test  of function Login test with: email, password
- Added navigate(instead of redirect to) for routes
- Protect routes (restring access if the user is not login)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,11 +46,13 @@ function App() {
...
         <Router>
-          {/* {user.username ? (
-              <Redirect from="/login" to="/" noThrow />
-            ) : (
-              <Redirect from="/" to="/login" noThrow />
-            )} */}
+          {currentUser.username ? (
+            <Redirect from="/login" to="/" noThrow />
+          ) : (
+            window.location.pathname !== "/login" && (
```
Comment (condef5) [inline src/index.js]: Hey, good aproach for protect all routes.

File: src/views/login.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 import { connect } from "react-redux";
-import { redirectTo } from "@reach/router";
+import { navigate } from "@reach/router";
 import { fetchLogin } from "../actions";
 
-function Login({ loading, getUser, currentUser, error }) {
-  const [email, setEmail] = React.useState("");
-  const [password, setPassword] = React.useState("");
+export function Login({ loading, getLogin, currentUser, error }) {
+  const [email, setEmail] = React.useState("josh@delivery.pe");
```
Comment (condef5) [inline src/views/login.js]: Hey, this is not necessary, could you change the `"josh@delivery.pe"` by `""` and `"123456"` by `""`

File: src/views/login.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import { Login } from "./login";
+
+test("Login View", () => {
+  const currentUser = {
```
Comment (condef5) [inline src/views/login.test.js]: The `currentUser` is not necessary for the init login, you could add another test for the redirect when a user is logged.
```
 <Login currentUser={null} loading={false} getLogin={getLogin} />
```

File: src/views/login.js (hunk 2 above)
Comment (AngieCristina) [reply src/views/login.js]: Thank you for letting me know , we must not have unnecessary code.

File: src/views/login.test.js (hunk 3 above)
Comment (AngieCristina) [reply src/views/login.test.js]: Great annotation! , thank you

**Pull request merged**